src/chrome/content/rules/ebaycommercenetwork.com.xml

```diff
@@ -0,0 +1,41 @@
+<!--
+	For other eBay coverage, see ebay.com.xml
+
+
+	Non-functional subdomains:
+		- $self		(r)
```

Comment:
same for `www`

Reply:
I agree and have now added "Mark https://www.ebaycommercenetwork.com as `Refused` in the top comment." to the checklist. Thanks.